kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/tools/kover/KoverHtmlOrXmlReport.kt

```diff
@@ -69,6 +70,10 @@ internal abstract class HtmlReportAction : AbstractReportAction<HtmlReportParame
         val files = parameters.files.get()
         val filters = parameters.filters.get()
 
+        // repeat reading freemarker temple from resources if error occurred, see https://github.com/Kotlin/kotlinx-kover/issues/510
+        // the values are selected empirically so that the maximum report generation time is not much more than a second
+        ReportApi.setFreemarkerRetry(7, 150)
```

Comment:
Do you plan to keep the issue open and eventually add a proper fix?

Reply:
If, after this fix, the error will still be reproduced, then it will be necessary to understand more deeply